install.sh

```diff
@@ -10,7 +10,7 @@ php -r "unlink('composer-setup.php');"
 sudo apt-get install nodejs npm
 
 # install npm gulp cli globally for css minification
-npm install -g gulp-cli
+sudo npm install -g gulp-cli
```

Comment:
@MehaKaushik  This will be helpful in installing dependencies easily. sudo is required for this specific command. If the complete script is executed using sudo, then directory ownership and read-write permissions will be affected (for directories like cache, vendor, node_modules etc). It's not a right way to run all commands using sudo. Only those commands which require sudo permissions should be executed with sudo permissions.